api_volontaria/apps/user/tests/tests_view_users.py

```diff
@@ -89,6 +85,54 @@ def test_profile(self):
             permissions
         )
 
+    def test_profile_admin(self):
+        self.client.force_authenticate(user=self.admin)
+        response = self.client.get(
+            'http://api.example.org/users/' + str(self.user.id)
+        )
```

Comment:
It's not really a test of personal profile since you get another user (line 91) and I'm afraid it will confuse people with the name of the test `test_profile` just before that really test the own profile of the user.

Maybe can you rename the test to something like `test_get_user_as_admin`